clang/lib/StaticAnalyzer/Checkers/SimpleStreamChecker.cpp

```diff
@@ -52,10 +52,13 @@ class SimpleStreamChecker : public Checker<check::PostCall,
                                            check::PreCall,
                                            check::DeadSymbols,
                                            check::PointerEscape> {
-  CallDescription OpenFn, CloseFn;
+  const CallDescription OpenFn{{"fopen"}, 2};
+  const CallDescription CloseFn{{"fclose"}, 1};
```

Comment:
Since we mostly use `CallDescription`s with compile time known inputs, it would be nice to make their ctors constexpr. This is not for this PR, just a note for the future.

Reply:
Well, it seems like `StringRef` is not constexpr for some reason - and I'm using that inside.
Consequently, that's a blocker.

Reply:
Here is why we can't have CallDescriptions constexpr:
It's an owning type, holding a `std::vector<std::string>`, and they are themselves non-constexpr, until IDK cpp20+?
So, this is resolved.